src/portal/modules/webmodule/backend/vision_on_edge/compute_devices/kan_client.py

```diff
@@ -18,6 +18,7 @@ def __init__(self):
         super().__init__()
         self.group = "fabric.kan"
         self.plural = "targets"
+        self.kan_api_url = "http://" + self.kan_ip + ":8080/v1alpha2/targets/registry"
```

Comment:
Can we make the port configurable as well? it's not necessarily on 8080. It could also be https instead of http.

Reply:
Sure, but how should we configure the port and protocol for http/https? Should it be selected in the installer or we need to get the value from the service?